static/app/views/replays/filters.tsx

```diff
@@ -9,17 +9,15 @@ import space from 'sentry/styles/space';
 
 function ReplaysFilters() {
   return (
-    <StyledPageFilterBar>
+    <StyledPageFilterBar condensed>
       <ProjectPageFilter />
       <EnvironmentPageFilter />
-      <DatePageFilter />
+      <DatePageFilter alignDropdown="left" />
```

Comment:
Isn't left default?

Reply:
not for the date page filter where when absolute date is selected it will align to the right.